hack/update_submodules.sh

```diff
@@ -123,30 +125,25 @@ if [ ! -z "$remote_branch" ]; then
   fi
 fi
 
-# Return if the working branch is dirty
-if ! git diff --quiet; then
-  echo "branch dirty, exiting to not overwrite work"
-  exit 1
-fi
 
-# For each branch that the user gives (master, release-4.7), create a new branch based off it,
-# with commits updating the submodules
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
Nice work, @wgahnagl. Just some early feedback... this sort of refactor should be done in a separate commit.